includes/wikia/models/DesignSystemSharedLinks.class.php

```diff
@@ -77,6 +77,7 @@ private function getLangWithFallback( $lang ) {
 			'app-store' => 'https://itunes.apple.com/developer/wikia-inc./id422467077',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "app-store" 11 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)